### ISSUE TITLE
NTR: Sepiapterin high in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8476]
New term request Sepiapterin high in CSF (segal_150721215042): Cerebrospinal fluid  UMLS: 2748714

**Comment by pnrobinson** [comment]
@MickeySegal
what is the disease context?

**Comment by MickeySegal** [comment]
https://pubmed.ncbi.nlm.nih.gov/22522443/
https://omim.org/entry/612716 "Laboratory analysis of CSF before treatment showed decreased HVA, 5-HIAA, and norepinephrine (3-methoxy-4-hydroxyphenylethyleneglycol; MHPG), and increased sepiapterin."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000975
- label: Elevated CSF sepiapterin concentration
- parents: [Abnormal CSF biopterin concentration (HP:0040207)](https://hpo.jax.org/app/browse/term/HP:0040207)
- pmid: PMID:18502672
- def: Concentration of sepiapterin in the cerebrospinal fluid (CSF) below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8476
- synonyms:
